packages/wonder-blocks-clickable/src/components/__tests__/clickable-behavior.test.js

```diff
@@ -21,6 +22,20 @@ const wait = (delay: number = 0) =>
         return setTimeout(resolve, delay);
     });
 
+const labelForState = (state: ClickableState): string => {
```

Comment:
**praise:** This is great! love how states got moved to the DOM so RTL can do what it does best!.

Reply:
Thankfully this was pretty easy to do since the component renders children.